index.d.ts

```diff
@@ -0,0 +1,23 @@
+declare module 'tftp-server' {
+    import dgram from "dgram";
+
+    export interface Request {
+        filename: string;
+        mode: "netascii" | "octet" | "mail"
+    }
+    export type Response = (buf: Buffer) => void
+    export type Handler = (req: Request, res: Response, next: () => void) => void
```

Comment:
The `next` callback has an optional parameter that takes an instance of `Error` to abort the handle lookup.

Reply:
Added :)